### DISCUSSION TITLE
Is there possible to implement multisite ?

### DISCUSSION THREAD

**Discussion by bachvtuan** [wintercms/winter#854] [Q&A] [answered]
Sharing same media files and login credentials but separate theme content, static pages, menu, etc.
Do you have any thought on this ?

**Comment by LukeTowers** [comment, marked as answer]
There is no one size fits all multisite solution, but in your specific case if you want to share everything except for themes then you can just use the domain driven environment configuration in order to set a different active theme per hostname: https://wintercms.com/docs/setup/configuration#domain-environment.

Basically you'd set the following in `config/environment.php`:

```php
<?php

return [
    'hosts' => [
        'sitea.website.tld' => 'sitea',
        'siteb.website.tld' => 'siteb',
    ]
];
```

then in `config/sitea/cms.php` you'd set the following:
```php
<?php

return [
    'activeTheme' => 'sitea',
];
```

and in `config/siteb/cms.php` you'd set the following:
```php
<?php

return [
    'activeTheme' => 'siteb',
];
```
